sphinx_gallery/tests/reference_parse.txt

```diff
@@ -10,13 +10,15 @@
  ('code',
   '\ndef dummy():\n    """This should not be part of a \'text\' block\'"""\n\n    ######################################\n    # Comment inside code to remain here\n    pass\n\n# this should not be part of a \'text\' block\n\n'),
  ('text',
-  '\n####################################################################\n\nMaking a line cut in sphinx\n'),
```

Comment:
The lstrip is removing the newline here. I am wondering whether it could it have an unintended consequence ... can you see any?